oauth2_http/java/com/google/auth/oauth2/OAuth2Credentials.java

```diff
@@ -163,6 +167,15 @@ public void refresh() throws IOException {
     }
   }
 
+  /**
+   * Provide additional headers to return as request metadata.
+   *
+   * @return Map of additional headers.
```

Comment:
nit: "map of additional headers" or just "additional headers" since the map is provided by the type

Reply:
Changed to "additional headers"